test/fastboot-response-test.js

```diff
@@ -8,11 +8,9 @@ describe("FastBootResponse", function() {
 
   beforeEach(function () {
     var mockResponse = {
-      headers: {
-        "i-am-a": "mock header, me too"
-      },
-      get: function () {
-        return this.headers.cookie;
+      _headers: {
```

Comment:
True, for a ServerResponse it is `this._headers`, for a ClientRequest it is `this.headers`.